gulp-tasks/utils/package-runner.js

```diff
@@ -2,12 +2,26 @@ const path = require('path');
 const glob = require('glob');
 const oneLine = require('common-tags').oneLine;
 
+const constants = require('./constants');
```

Comment:
Sorry I'm not a fan of this approach.

Could we just have two variants of package runner. One for node and one for browser, and depending on the package runner called, the func is called with the relevant packages.

```js
browserPackageRunner(displayName, func)
nodePackageRunner(displayName, func)
```

This change along with seperating browser builds from node builds in gulp tasks should make things a little cleaner.

Reply:
There's still an underlying question in v3 of how we determine whether a given package targets the browser or Node—in v2, when we shipped build rules and tests in each package's directory, it wasn't ambiguous.

I'm cool with renaming the tasks and breaking them up into separate files, but I think the lingering question is whether you're on board with checking `engines.node` in `package.json` and using that to determine whether we're in Browser or Node "mode".